backend/src/Types/Database.hs

```diff
@@ -80,6 +80,15 @@ migrateSchema dbPool logger = do
   unless (null migrations) (log logger LevelWarn "Database schema changed. Running migrations.")
   mapM_ (log logger LevelDebug . toLogStr) migrations
 
+  let index_statements =
```

Comment:
Unfortunately, the Persistent DSL doesn't handle index creation, so we have to use raw/untyped SQL here.

Reply:
🤮 (joking, this is nbd lol)